### PR TITLE
Re-use landuse sort order.

### PR DESCRIPTION
**Pull request by zerebubuth**
Previously, the sort order of the keys had to be passed in directly from the configuration. This would have meant duplicating the order in two places, so it's better that we can re-use the order parameter in the sort.

@rmarianski could you review, please?

Sort-of refs mapzen/vector-datasource#198.

### CODE REVIEW

File: TileStache/Goodies/VecTiles/transform.py [hunk 1]
```diff
@@ -385,6 +385,63 @@ def tags_name_i18n(shape, properties, fid, zoom):
...
+    in the right format, attrs isn't a dict then returns
+    None.
+    """
+
+    sort_key = attrs.get('sort_key')
+    reverse = attrs.get('reverse')
+
+    assert sort_key is not None, "Configuration " + \
+        "parameter 'sort_key' is missing, please " + \
+        "check yout configuration."
```
Comment (rmarianski) [inline TileStache/Goodies/VecTiles/transform.py]: spelling: yout -> your

File: TileStache/Goodies/VecTiles/transform.py [hunk 2]
```diff
@@ -385,6 +385,63 @@ def tags_name_i18n(shape, properties, fid, zoom):
...
+
+    # extract the sorted list of attributes from the
+    # grouped (attribute, order) pairs, ordering by
+    # the order.
+    all_attrs = sorted(group.iteritems(),
+        key=lambda x: x[1])
+
+    # if we wanted the sort reversed, then reverse it
+    if reverse:
+        all_attrs = reversed(all_attrs)
```
Comment (rmarianski) [inline TileStache/Goodies/VecTiles/transform.py]: Should probably combine with the above:

``` python
all_attrs = sorted(group.iteritems(), key=lambda x: x[1], reverse=reverse)
```

**Comment by rmarianski** [comment]
:+1:

File: TileStache/Goodies/VecTiles/transform.py (hunk 1 above)
Comment (zerebubuth) [inline TileStache/Goodies/VecTiles/transform.py]: Wel spotted! Thanks!

File: TileStache/Goodies/VecTiles/transform.py (hunk 2 above)
Comment (zerebubuth) [inline TileStache/Goodies/VecTiles/transform.py]: Oh, nice! I didn't know `sorted` had that.

**Pull request merged**